.github/workflows/build.yml

```diff
@@ -15,7 +15,7 @@ jobs:
         with:
           node-version: '18'
           cache: 'npm'
```

Comment:
It's worth changing this to `cache: 'yarn'` as a result of the changes in this PR.

(Sorry for the drive-by commenting, instead of doing a PR)